lab-jonathan/index.js

```diff
@@ -0,0 +1,7 @@
+var greet = require('./lib/greet.js');
+
+
+var name = process.argv[2];
+
+
+console.log(greet(name));
```

Comment:
At this point you can even pass `process.argv[2]` directly into the greet method.